archlinuxcn/kwin-decoration-sierra-breeze-enhanced/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: A1ca7raz <aya@wtm.moe>
+
+pkgname=kwin-decoration-sierra-breeze-enhanced
+_gitname=SierraBreezeEnhanced
+pkgver=2.0.1
+pkgrel=1
+pkgdesc="Originally a fork of BreezeEnhanced to make it (arguably) more minimalistic and informative"
+arch=('x86_64')
+url="https://github.com/kupiqu/SierraBreezeEnhanced"
+license=('GPL-3.0-only')
+depends=('kwin')
+makedepends=('cmake' 'extra-cmake-modules')
+source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/refs/tags/V${pkgver}.tar.gz")
+sha256sums=('cc040c49ab5bb9a073988b8e9c4417371713c3cd0366af16cc47b7cfdcd4216e')
+
+build() {
+    cd "${srcdir}/${_gitname}-${pkgver}"
+
+    mkdir build && cd build
+    cmake .. \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DKDE_INSTALL_LIBDIR=lib \
+        -DBUILD_TESTING=OFF \
+        -DKDE_INSTALL_USE_QT_SYS_PATHS=ON
+}
+
+package() {
+    make -C ${srcdir}/${_gitname}-${pkgver}/build DESTDIR="$pkgdir" install
+}
```

Comment:
When building packages with cmake, I would prefer to use cmake's built-in commands:

```bash
# create build dir and generate build files
cmake -B build -S "${srcdir}/${_gitname}-${pkgver}"

# compile the source
cmake --build build

# install
DESTDIR="${pkgdir}" cmake --install build
```

minimal example:

https://github.com/archlinuxcn/repo/blob/2a8657c63f0b4958fe767ee616d48ad005b32081/archlinuxcn/ksysguard6-git/PKGBUILD#L42-L49